ImageNet/train/huaweicloud_resnet50_modelarts_16_machine_mxnet121.json

```diff
@@ -0,0 +1,43 @@
+{
+    "version": "v1.0",
+    "author": "ModelArts Service of Huawei Cloud",
+    "authorEmail": "ModelArts Service of Huawei Cloud",
+    "framework": "MXNet v1.2.1",
+    "codeURL": "https://github.com/huawei-clouds/modelarts-example/blob/eb04ab2830814a70667210ee97c04ddb2aaa0059/MXNet-ImageNet-10min/16_32_dysizelr.py",
+    "model": "ResNet-50",
+    "hardware": "16*8*Tesla-V100(ModelArts Service)",
+    "timestamp": "2018-11-27",
+    "misc": {
+        "model implementation": "https://github.com/apache/incubator-mxnet/blob/251ae71a20d8318ab20f4c19520f74b881fdf3ff/example/image-classification/symbols/resnet-v1.py",
+        "backend": "16 nodes (8*V100 for each node)",
+        "CUDA version": "9.0",
+        "CuDNN version": "7.0.2",
+        "optimizer": "NAG",
+        "Initializer": "MSRAPrelu",
+        "momentum": 0.9,
+        "weightDecay": 0.0001,
+        "schedule": [
+            {"ep":0, "lr":1.92, "mode":"warmup"},
+            {"ep":6, "lr":3.84, "mode":"node"},
+            {"ep":11, "lr":3.84, "mode":"node"},
+            {"ep":14, "lr":3.84, "mode":"linear"},
+            {"ep":17, "lr":1.92, "mode":"cosine"},
+            {"ep":32, "lr":0.0192, "mode":"node"}
+        ],
+
+        "imageSize": [
+            {"image_size": 128, "epochs": 14},
+            {"image_size": 224, "epochs": 14},
+            {"image_size": 288, "epochs": 4}
+        ],
+
+        "batchSize": [
+            {"batch_size": 8192, "epochs": 6},
+            {"batch_size": 16384, "epochs": 8},
+            {"batch_size": 16384, "epochs": 3},
+            {"batch_size": 20480, "epochs": 11},
+            {"batch_size": 16384, "epochs": 4}
+        ]
+    },
+    "usedBlacklist": false
```

Comment:
Ok, please check it again.
We just revalidated the last three epoch.

Reply:
@TccccD would it be possible to revalidate on all the checkpoints?

Reply:
Since the MXNet platform cannot continuously bind the model for verification(out of memory), we ran it again, with slightly different time and precision.
Please check it again. @codyaustun

Reply:
@TccccD thanks!